script/PR_external_chart_template.md

```diff
@@ -0,0 +1,27 @@
+Signed-off-by: <USER> <EMAIL>
+
+**Description of the change**
+
+This is an automatic PR for synchronizing the changes performed externally at the [kubeapps development chart](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps) official chart.
+
+**Benefits**
+
+The Kubeapps [bitnami/kubeapps chart](https://github.com/bitnami/charts/tree/master/bitnami/kubeapps) will get the newest changes, including image version updates and other updates.
+
+**Possible drawbacks**
+
+Even unlikely, some changes implemented by the Kubeapps team may not be 100% compatible with the current chart version.
```

Comment:
We should never be pushing new kubeapps application images that change functionality based on chart input without a new chart release I think? I don't understand when this PR would be triggered unless when we're wanting to release a new chart version?
```suggestion
Although unlikely, some changes implemented by the Kubeapps team may not be 100% compatible with the current chart version.
```